playwright-e2e/dsm/pages/participant-list-page.ts

```diff
@@ -362,9 +363,20 @@ export default class ParticipantListPage extends DsmPageBase {
           if (!participantData || participantData[0] === undefined) {
             continue;
           }
-          shortID = JSON.stringify(value.esData.profile.hruid).replace(/['"]+/g, '');
-          logInfo(`Found RGP participant with Short ID: ${shortID} to have proband tabs`);
-          return shortID;
+          const familyMemberInfo = value.participantData;
+          const amountOfFamilyMembers = familyMemberInfo.length;
+          if (rgpMinimumFamilySize > 1 && amountOfFamilyMembers === 1) {
+            continue; //Looking for a family that has more than 1 person in the study
+          } else if (rgpMinimumFamilySize > 1 && amountOfFamilyMembers > 1) {
+            //Use the family member added to the account after the proband
+            shortID = JSON.stringify(value.esData.profile.hruid).replace(/['"]+/g, '');
+            logInfo(`Found RGP participant with Short ID: ${shortID} to have proband tabs - has ${amountOfFamilyMembers} study participants`);
+            return shortID;
+          } else if (rgpMinimumFamilySize === 1 && amountOfFamilyMembers === 1) {
+            shortID = JSON.stringify(value.esData.profile.hruid).replace(/['"]+/g, '');
+            logInfo(`Found RGP participant with Short ID: ${shortID} to have proband tabs - has 1 study participant`);
+            return shortID;
```

Comment:
I think you can move the `return shortID` down to the bottom of the conditional block and avoid repeating it here and on L373.